src/Core/UpdateManager.vala

```diff
@@ -25,9 +25,10 @@ public class AppCenterCore.UpdateManager : Object {
     public signal void installed_apps_changed ();
     public signal void cache_update_failed (Error error);
 
+    public ListStore updates_liststore { public get; private set; }
     public Package runtime_updates { public get; private set; }
     public int unpaid_apps_number { get; private set; default = 0; }
-    public uint updates_number { get; private set; default = 0U; }
+    public uint updates_number { get; set; default = 0U; }
```

Comment:
Absolutely not necessary but to make it even more expressive you could do 
```suggestion
    public uint updates_number { get { return updates_liststore.n_items; } }
```

Reply:
I think it has to be settable because there are things that bind to this property so they need the set notify signal